funnel/views/email.py

```diff
@@ -139,6 +147,37 @@ def send_email_for_organization_admin_membership_revoked(
     )
 
 
+@signals.user_registered_for_project.connect
+def send_email_for_project_registration(project, user, locale=None):
+    background_project_email.queue(
+        project_id=project.id, user_id=user.id, locale=get_locale()
+    )
+
+
+@rq.job('funnel')
+def background_project_email(project_id, user_id, locale=None):
+    EmailAttachment = namedtuple('EmailAttachment', ['file', 'filename', 'mimetype'])
```

Comment:
`namedtuple` creates classes. The camel casing of `EmailAttachment` is the hint. Create this class at the top of this file, right after imports.